Calorie Tracker/Calorie Tracker/ViewController.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ViewController.swift
+//  Calorie Tracker
+//
+//  Created by Alex Thompson on 2/22/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
```

Comment:
Before turning the sprint in, you should double check to see if you can removes unnecessary files.